tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -54,6 +54,18 @@ def get_label_params(resource, all_labels):
                         'labelFingerprint': resource['labelFingerprint']
                     }}
 
+        @staticmethod
+        def refetchFingerprint(params, client, model, resource):
+            try:
+                resource = model.get(client,{
+                    'project_id': params['project'],
```

Comment:
hmm.. not sure why this is being flagged as not covered, the tests do cover afaics.